.pylintrc

```diff
@@ -0,0 +1,935 @@
+        format-string-without-interpolation,
+        anomalous-backslash-in-string,
+        anomalous-unicode-escape-in-string,
+        implicit-str-concat,
+        inconsistent-quotes,
+        redundant-u-string-prefix,
+        bad-open-mode,
+        boolean-datetime,
+        redundant-unittest-assert,
+        deprecated-method,
+        bad-thread-instantiation,
+        shallow-copy-environ,
+        invalid-envvar-default,
+        subprocess-popen-preexec-fn,
+        subprocess-run-check,
+        deprecated-argument,
+        deprecated-class,
+        deprecated-decorator,
+        unspecified-encoding,
+        forgotten-debug-statement,
+        apply-builtin,
+        basestring-builtin,
+        buffer-builtin,
+        cmp-builtin,
+        coerce-builtin,
+        execfile-builtin,
+        file-builtin,
+        long-builtin,
+        raw_input-builtin,
+        reduce-builtin,
+        standarderror-builtin,
+        unicode-builtin,
+        xrange-builtin,
+        coerce-method,
+        delslice-method,
+        getslice-method,
+        setslice-method,
+        no-absolute-import,
+        old-division,
+        dict-iter-method,
+        dict-view-method,
+        next-method-called,
+        metaclass-assignment,
+        indexing-exception,
+        raising-string,
+        reload-builtin,
+        oct-method,
+        hex-method,
+        nonzero-method,
+        cmp-method,
+        input-builtin,
+        round-builtin,
+        intern-builtin,
+        unichr-builtin,
+        map-builtin-not-iterating,
+        zip-builtin-not-iterating,
+        range-builtin-not-iterating,
+        filter-builtin-not-iterating,
+        using-cmp-argument,
+        eq-without-hash,
+        div-method,
+        idiv-method,
+        rdiv-method,
+        exception-message-attribute,
+        invalid-str-codec,
+        sys-max-int,
+        bad-python3-import,
+        deprecated-string-function,
+        deprecated-str-translate-call,
+        deprecated-itertools-function,
+        deprecated-types-field,
+        next-method-defined,
+        dict-items-not-iterating,
+        dict-keys-not-iterating,
+        dict-values-not-iterating,
+        deprecated-operator-function,
+        deprecated-urllib-function,
+        xreadlines-attribute,
+        deprecated-sys-function,
+        exception-escape,
+        comprehension-escape
+
+# Enable the message, report, category or checker with the given id(s). You can
+# either give multiple identifier separated by comma (,) or put this option
+# multiple time (only on the command line, not in the configuration file where
+# it should appear only once). See also the "--disable" option for examples.
+enable=superfluous-parens,
+       multiple-imports,
+       wrong-import-order,
+       ungrouped-imports,
+       wrong-import-position,
+       useless-import-alias,
+       cyclic-import,
+       reimported,
+       import-self,
+       misplaced-future,
+       unused-import
```

Comment:
While I like seeing the enabled rules, is this necessary? I would've assumed that these enabled simply by not being disabled.

Reply:
Not sure. I converted the CLI args to a file using a CLI flag. Did not write this file manually.

Reply:
`pylint -s n --disable=all --enable=C0325,C0410,C0411,C0412,C0413,C0414,W0611,R0401,W0404,W0406,W0410 --generate-rcfile > .pylintrc`

Reply:
Rules that are not listed are automatically enabled I think. But this may be nice to help us keep track of which items we've fixed in the code-base.

Reply:
Sounds good!